repositories/user-repository.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-return-await */
+/* eslint-disable arrow-parens */
+
+const { User } = require('../models');
+
+module.exports = {
+  findAllUsers: async () =>
+    await User.findAll({ attributes: { exclude: ['id', 'password'] } }),
+
+  findUserByPk: async (id) => await User.findByPk(id),
+
+  findUserByEmail: async (email) => await User.findOne({ where: { email } }),
+
+  createUser: async (attributes) => await User.create(attributes),
```

Comment:
I suggest replacing the `create` method with `findOrCreate` in order to avoid creating multiple users with the same email (the model and database already specify a UNIQUE email anyway, this would only be an extra layer of logic)